.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
           keys:
             - omnibus-rpm-cache-{{ checksum "~/cache_key.txt" }}
       - run: echo -e $RPM_GPG_KEY > /home/circleci/keys/RPM-SIGNING-KEY.private
-      - run: docker run -e RELEASE_VERSION=$RELEASE_VERSION -e RPM_SIGNING_PASSPHRASE=$RPM_SIGNING_PASSPHRASE -e OMNIBUS_BRANCH=$OMNIBUS_BRANCH -v /home/circleci/keys:/keys -v /home/circleci/pkg:/dd-agent-omnibus/pkg -v /home/circleci/cache:/var/cache/omnibus datadog/docker-dd-agent-build-rpm-x64
+      - run: docker run -e RELEASE_VERSION=$RELEASE_VERSION -e RPM_SIGNING_PASSPHRASE=$RPM_SIGNING_PASSPHRASE -e OMNIBUS_BRANCH=$OMNIBUS_BRANCH -v /home/circleci/keys:/keys -v /home/circleci/pkg:/dd-agent-omnibus/pkg -v /home/circleci/cache:/var/cache/omnibus jfullaondo/docker-dd-agent-build-rpm-x64:latest
```

Comment:
Note to reviewers: Before merging this will be changed back and reverted to `datadog/docker-dd-agent-build-rpm-x64`. 

This is currently pointing to my own private repository to test 
(successfully) the build pipeline.

We will also need to push the image resulting from the Dockerfile changes below to the dockerhub `datadog/docker-dd-agent-build-rpm-x64` for the agent5 pipeline to find the correct image.